local_xy_origins: [{name: swri, latitude: 41.4725444511, longitude: 1.96027490222, altitude: 91.7481719619, heading: 0}]